src/GafferImage/Checkerboard.cpp

```diff
@@ -57,12 +57,8 @@ namespace
 
 inline float filteredStripes( float x, float period, float filterWidth )
 {
-	// \todo : round / floor and int conversion are actually pretty slow by default.
-	// Could be worth trying out some intrinsics here, though of course, if we actually
-	// wanted this to not be slow, having a separable code path when there is no rotation
-	// would be a much bigger speedup
 	float xp = x / ( period * 0.5f );
-	float nearestBoundary = round( xp );
+	float nearestBoundary = nearbyintf( xp );
```

Comment:
This is a new one to me, so I had to check the docs. They seem to say that the result is affected by the current rounding mode, but do we have any guarantee that the current mode is `FE_TONEAREST`?

Reply:
Reverted this change, added comment explaining options in case we ever revisit this.